src/iosource/Manager.cc

```diff
@@ -255,11 +255,11 @@ void Manager::Register(IOSource* src, bool dont_count)
 	{
 	// First see if we already have registered that source. If so, just
 	// adjust dont_count.
-	for ( SourceList::iterator i = sources.begin(); i != sources.end(); ++i )
+	for ( auto iosrc : sources )
```

Comment:
style nit: for such loops I would suggest using either `const auto&` (if not modifying) or `auto&` if modifying, just for clarity, even though it doesn't make a performance difference here because it's pointers anyways.

Reply:
Fixed and force pushed.